README.md

```diff
@@ -15,20 +15,35 @@ administrative privileges (`sudo`).
 
 If you have obtained the project source from the __version control
 repository__, you'll need to have GNU Build System (Autotools) installed.
-In Debian/Ubuntu based platforms, you may install the required software with:
+
+### Debian/Ubuntu based platforms:
+
+ ```
+ $ apt install automake autoconf
```

Comment:
It's actually `#`, to indicate it needs `sudo`. Same for line 38 and (maybe) for 25 and 44.